test/e2e/kuttl/fsconfig-file-sync/00-install.yaml

```diff
@@ -0,0 +1,129 @@
+# This configuration deploys the means to test the mutating injection webhook by proxying through an nginx server
+# to assert that flagd is reachable
+---
+apiVersion: core.openfeature.dev/v1alpha3
+kind: FlagSourceConfiguration
+metadata:
+    name: source-configuration
+spec:
+    metricsPort: 8080
+    evaluator: json
+    defaultSyncProvider: filepath
+    tag: v0.5.0
+    sources:
+    - source: end-to-end-test
+      provider: filepath
+    probesEnabled: true
+---
+apiVersion: core.openfeature.dev/v1alpha1
```

Comment:
Is it possible or desirable to extract this resource instead of repeating it? I'm not sure what the best style is with KUTTL tests. I don't mind too much either way, just thought.

Reply:
Not sure to be honest, but will definitely look into this deeper.

Just a brief explanation how KUTTL works: Kuttl creates a separate namespace for each test (one test == one directory) and starts to apply the *-install.yaml files with basic `kubectl` command. So it starts with `00-install.yaml`. Afterwards it waits until the conditions in `00-assert.yaml` are met (it compares the rows of the resource in *-asser.yaml with the exisitng resource in cluster). When the conditions are met, it continues in the same way with 01-install.yaml and 01-assert.yaml. On the other side, if the conditions is not met until the timeout (I think 5min by default), test fails.

The good side of this approach (have the same resources multiple times) is that we always see what gets applied in the cluster. But I also see the problem with resource duplications. Will definitely search if there is a nicer way for it :+1: